src/main/java/task/NewOpportunityPortal/service/impl/MessageServiceImpl.java

```diff
@@ -16,20 +23,39 @@
 public class MessageServiceImpl implements MessageService{
 
     private final MessageRepository repository;
+    private final EncryptDecrypt encryptDecrypt;
 
     @Override
     public Long createMessage(Message message) {
         Date now = new java.util.Date();
         log.info("Set time creates: {}",  now);
         message.setCreateAt(new java.sql.Timestamp(now.getTime()));
+        try {
+            log.info("Encoder message");
+            message.setText(encryptDecrypt.encrypt(message.getText()));
+        } catch (BadPaddingException | IllegalBlockSizeException e) {
+            log.error(e.getMessage());
```

Comment:
pass full exception into the exception. Then you will get full stack trace in the log and it will be easier to understand what cause this issue.